### ISSUE TITLE
WAI-ARIA group is missing a name

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/report-a-cybercrime#170]
**WAI-ARIA group is missing a name**
 1.3.1 Info and Relationships
A group created with the WAI-ARIA role="group" or role="radiogroup" is missing a name.

Use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to name the group.

Code

*<div style="outline: none; z-index: 2147483646; position: relative;" tabindex="-1" role="group"><div class="css-9u32ac" style=""><h1 class="css-11o3zhs e76bb90" style="">Have you or someone you know encountered a cybercrime?</h1><h3 class="css-1dfj01i e76bb91" style="">Tell us your story in three easy steps:</h3><li class="e1vcssft0 css-16c26gp-StyledListItem-StyledHoc e4u1u2g0" style="">Describe what happened.</li> <li class="e1vcssft0 css-16c26gp-StyledListItem-StyledHoc e4u1u2g0" style="">Select where you encountered the cybercrime.</li> <li class="e1vcssft0 css-16c26gp-StyledListItem-StyledHoc e4u1u2g0" style="">Share how you were impacted.</li><div class="e1vcssft0 css-kby5jq-StyledWarningText-StyledHoc e4u1u2g0" style=""><div class="css-96kz36-IconImportantWrapper e4u1u2g1" style=""><svg version="1.1" fill="currentColor" width="100%" height="100%" style="display: block;" viewBox="0 0 35.000000 35.000000" preserveAspectRatio="xMidYMid meet"><title style="transition: none 0s ease 0s;">icon important</title><g transform="translate(0.000000,35.000000) scale(0.100000,-0.100000)" style=""><path d="M100 332 c-87 -48 -125 -155 -82 -232 48 -87 155 -125 232 -82 87 48
125 155 82 232 -48 87 -155 125 -232 82z m100 -122 c0 -53 -2 -60 -20 -60 -18
0 -20 7 -20 60 0 53 2 60 20 60 18 0 20 -7 20 -60z m0 -111 c0 -12 -7 -19 -20
-19 -19 0 -28 28 -14 43 11 11 34 -5 34 -24z" style=""></path></g></svg></div><strong class="css-jegmsb-WarningTextWrapper e4u1u2g2" style="">Please do not provide any personal information.</strong></div><a class="css-1hldnp" href="/form1" style="">Share your story→</a><div class="css-15uyi5f" style=""><a href="http://www.rcmp-grc.gc.ca/en/cybercrime-an-overview-incidents-and-issues-canada" style="">What is a cybercrime?</a></div></div></div>*

**How do I meet this criteria?**
Use one of the following techniques from WCAG 2:

ARIA17 Using grouping roles to identify related form controls
These references are taken from the Web Content Accessibility Guidelines 2.0.

<img width="1431" alt="screen shot 2019-03-06 at 5 49 06 pm" src="https://user-images.githubusercontent.com/35537308/53919512-39962c00-4038-11e9-96ed-cb36be2a2321.png">

**Comment by sastels** [comment]
This div / role is added by `reach-router` to improve accessibility - we're probably using it wrong.

**Comment by sastels** [comment]
This code has since been deleted. We should run a new accessibility assessment on the current prototype.

(mind you, it's quite possible that the current code has this issue)
